src/scenes/UserGalleryPage/UserGalleryHeader.tsx

```diff
@@ -1,6 +1,6 @@
 import { useMemo } from 'react';
 import styled from 'styled-components';
-import unescape from 'lodash.unescape';
+import unescape from 'src/utils/unescape';
```

Comment:
you can toss `src/` and just straight import from `utils/unescape`!

<img width="470" alt="image" src="https://user-images.githubusercontent.com/12162433/159389399-bb0d3a64-6b53-4764-a297-220a482719c5.png">